### DISCUSSION TITLE
Welcome to pashmak Discussions!

### DISCUSSION THREAD

**Discussion by parsampsh** [pashmaklang/pashmak#21] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mehanalavimajd** [comment]
The update progress has really been slow this year.
Are you still maintaining the repo?

**Reply by parsampsh** [reply]
Hi. Actually I have been a bit busy. But still maintaining the repo. I may add some new features and release 0.9. But i don't think i be able to do it now. It may take some time...
